### ISSUE TITLE
Thought: Test centre can be vaccine centres?

### ISSUE THREAD

**Issue by juanfranblanco** [Nethereum/Covid19-Immunity-Cert-Attestation#11]
@jaerith as we want vaccines to be distributed equally, could this not be used as well to find a vaccine centre? Obviously it could be a pop up, then pure guess on procedure after vaccination a test might be required for immunity and release the certificate?

**Comment by jaerith** [comment]
@juanfranblanco Yes, when vaccines do become available, they could be distributed via sample centres or test centres.  Of course, in some cases, sample centres and test centres could be one and the same, but I imagine that those would be in the minority.  However, at the point when the vaccine has not yet reached mass production, I could see the results from a testing centre determining the prioritization of who gets the vaccine first.  So, if Bob and Alice get their results and it's determined that Bob has antibodies while Alice has none, Alice would receive the vaccine before Bob.
